docs/scos/dev/technology-partner-guides/202204.0/payment-partners/crefopay/integrating-crefopay.md

```diff
@@ -829,3 +829,12 @@ class PaymentDependencyProvider extends SprykerPaymentDependencyProvider
     }
 }
 ```
+
+{% info_block warningBox "Note" %}
+
+In case an additional validation of input fields that should be filled by a customer is needed, the recommended way is to create a plugin that implements `\Spryker\Zed\CheckoutExtension\Dependency\Plugin\CheckoutPreConditionPluginInterface`.
+The plugin should be added to the `getCheckoutPreConditions` method of `Pyz\Zed\Checkout\CheckoutDependencyProvider.php`.
```

Comment:
```suggestion
The plugin should be added to the `getCheckoutPreConditions` method of `Pyz\Zed\Checkout\CheckoutDependencyProvider`.
```

Reply:
could be simplified to `Pyz\Zed\Checkout\CheckoutDependencyProvider::Pyz\Zed\Checkout\CheckoutDependencyProvider`